TOC.md

```diff
@@ -2,379 +2,379 @@
-      - [Table Filter](dev/reference/tools/tidb-lightning/table-filter.md)
-      - [CSV Support](dev/reference/tools/tidb-lightning/csv.md)
-      - [Monitor](dev/reference/tools/tidb-lightning/monitor.md)
-      - [Troubleshoot](dev/how-to/troubleshoot/tidb-lightning.md)
-      - [FAQ](faq/tidb-lightning.md)
+      - [Overview](/dev/reference/tools/tidb-lightning/overview.md)
+      - [Deployment](/dev/reference/tools/tidb-lightning/deployment.md)
+      - [Checkpoints](/dev/reference/tools/tidb-lightning/checkpoints.md)
+      - [Table Filter](/dev/reference/tools/tidb-lightning/table-filter.md)
+      - [CSV Support](/dev/reference/tools/tidb-lightning/csv.md)
+      - [Monitor](/dev/reference/tools/tidb-lightning/monitor.md)
+      - [Troubleshoot](/dev/how-to/troubleshoot/tidb-lightning.md)
+      - [FAQ](/dev/faq/tidb-lightning.md)
     + TiDB Binlog
-      - [Overview](dev/reference/tools/tidb-binlog/overview.md)
-      - [Deploy](dev/reference/tools/tidb-binlog/deploy.md)
-      - [Monitor](dev/reference/tools/tidb-binlog/monitor.md)
-      - [Maintain](dev/reference/tools/tidb-binlog/operation.md)
-      - [Upgrade](dev/reference/tools/tidb-binlog/upgrade.md)
-    - [PD Control](dev/reference/tools/pd-control.md)
-    - [PD Recover](dev/reference/tools/pd-recover.md)
+      - [Overview](/dev/reference/tools/tidb-binlog/overview.md)
+      - [Deploy](/dev/reference/tools/tidb-binlog/deploy.md)
+      - [Monitor](/dev/reference/tools/tidb-binlog/monitor.md)
+      - [Maintain](/dev/reference/tools/tidb-binlog/operation.md)
+      - [Upgrade](/dev/reference/tools/tidb-binlog/upgrade.md)
+    - [PD Control](/dev/reference/tools/pd-control.md)
+    - [PD Recover](/dev/reference/tools/pd-recover.md)
     - [TiKV Control](https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md)
-    - [TiDB Control](dev/reference/tools/tidb-control.md)
-    - [Download](dev/reference/tools/download.md)
+    - [TiDB Control](/dev/reference/tools/tidb-control.md)
+    - [Download](/dev/reference/tools/download.md)
   + Key Monitoring Metrics
-    - [Overview](dev/reference/key-monitoring-metrics/overview.md)
-    - [TiDB](dev/reference/key-monitoring-metrics/tidb.md)
-    - [PD](dev/reference/key-monitoring-metrics/pd.md)
-    - [TiKV](dev/reference/key-monitoring-metrics/tikv.md)
-  - [Adopters](adopters.md)
+    - [Overview](/dev/reference/key-monitoring-metrics/overview.md)
+    - [TiDB](/dev/reference/key-monitoring-metrics/tidb.md)
+    - [PD](/dev/reference/key-monitoring-metrics/pd.md)
+    - [TiKV](/dev/reference/key-monitoring-metrics/tikv.md)
+  - [Adopters](/dev/adopters.md)
 + FAQs
-  - [TiDB FAQs](faq/tidb.md)
-  - [TiDB Lightning FAQs](faq/tidb-lightning.md)
-  - [Upgrade FAQs](faq/upgrades.md)
-+ Support  
-  - [Support Resources](support-resources.md)
-  - [Report an Issue](report-issue.md)
+  - [TiDB FAQs](/dev/faq/tidb.md)
+  - [TiDB Lightning FAQs](/dev/faq/tidb-lightning.md)
+  - [Upgrade FAQs](/dev/faq/upgrades.md)
```

Comment:
This file name is changed to `upgrade` to make it consistent with that in docs-cn (https://github.com/pingcap/docs/pull/1209). I think `upgrade` is enough to convey the meaning.

Reply:
Fixed.